frontend/apps/reader/modules/readertypeset.lua

```diff
@@ -506,13 +506,19 @@ function ReaderTypeset:onSetPageMargins(margins, when_applied_callback)
             text = T(_([[
 Margins set to:
 
-  left: %1 (%2px)
-  right: %3 (%4px)
-  top: %5 (%6px)
-  bottom: %7 (%8px)
+  left: %1
+  right: %2
+  top: %3
+  bottom: %4
+
+  footer: %5 px
 
 Tap to dismiss.]]),
-            margins[1], left, margins[3], right, margins[2], top, margins[4], bottom),
+            optionsutil.formatFlexSize(margins[1]),
+            optionsutil.formatFlexSize(margins[2]),
+            optionsutil.formatFlexSize(margins[3]),
+            optionsutil.formatFlexSize(margins[4]),
+            self.view.footer.reclaim_height and 0 or self.view.footer:getHeight()),
```

Comment:
No flex size for the footer height ?

Reply:
Nope, it's already in pixels ;)).

Reply:
Like the margins, but we show them in cm/in - so why not the footer height?

Reply:
The variables being used for the margins here is pre-scaling, so they're still flex, not absolute pixels like the footer:

https://github.com/NiLuJe/koreader/blob/bfecf88f38271131d3fde701f68be0d169ff3011/frontend/apps/reader/modules/readertypeset.lua#L490-L498